templates/content/blank/.template.config/template.json

```diff
@@ -15,7 +15,7 @@
     "sourceName": "NewApp",
     "primaryOutputs": [
         {
-            "path": "NewApp\\NewApp.fs"
+            "path": "NewApp\\NewApp.fsproj"
```

Comment:
When creating a new project in VS for Mac, the core project wasn't created until I changed this. Not sure on the impact on dotnet new though.

Reply:
ok, should be ok